.circleci/config.yml

```diff
@@ -286,6 +289,7 @@ jobs:
                   $libratest $target --test-threads 1 --exact --nocapture
                 status=$?
                 retry=$((retry + 1))
+                sleep 10
```

Comment:
not sure why the sleep 10?

Reply:
seems safe -- probably to give background processes time to die?

Reply:
Yep. It's insurance.